README.md

```diff
@@ -31,46 +37,39 @@ Further information is included in the following documents:
 - [Oak Functions](oak_functions/README.md) presents our computing platform for
   developing stateless applications in a privacy preserving way.
 
-## Terminology
+## Parties involved
 
-- **Enclave**: A secure CPU compartment that can be created on-demand,
-  containing code and data; it enforces isolation from the host and other
-  enclave instances running on the same system. It guarantees _confidentiality_
-  and _integrity_ of both data and code running within it, and it is capable of
-  creating hardware-backed _remote attestations_ to prove to other parties a
-  measurement (i.e. hash) of the code and data within the enclave itself. Also
-  known as Trusted Execution Environment (TEE).
-- **Enclave Manufacturer**: The entity in charge of manufacturing the CPU or
-  System on a Chip (SoC) supporting enclaves.
+- **Oak Trusted Runtime Authors**: The authors of the code in this repository;
+  mostly this corresponds to the Project Oak team, but also any contributors,
+  and, by extension, the authors of third party dependencies used in Oak.
+- **TEE Manufacturer**: The entity in charge of manufacturing the CPU or System
+  on a Chip (SoC) supporting the TEE, including hardware, software, and
+  cryptographic keys.
 - **Platform Provider**: The entity in charge of maintaining and running the
-  combined hardware and software stack surrounding the TEE, for instance in a
-  cloud context.
-- **Trusted Computing Base (TCB)**: The set of hardware, firmware, software
-  components critical to the security of the system; bugs or vulnerabilities
-  inside the TCB may jeopardise the security properties of the entire system.
-- **Independent Software Vendor (ISV)**: The entity or person providing the code
-  for the service running on top of the Project Oak; in the most common case
-  this may be a third party developer.
+  combined hardware and software stack surrounding the TEE, for instance a cloud
+  provider; this includes their software, hardware, and employees.
+- **Untrusted Business Logic Authors**: The authors of the untrusted business
+  logic, compiled to Wasm, that is executed by the Oak Functions Runtime.
 
 ## Threat Model
 
 - **untrusted**:
   - most hardware (memory, disk, motherboard, network card, external devices)
   - Operating System (kernel, drivers, libraries, applications)
-  - platform provider (hardware, software, employees)
-  - third-party developers
+  - Platform Provider
+  - Untrusted Business Logic Authors
 - **trusted-but-verifiable**:
-  - Project Oak codebase (and its transitive dependencies)
+  - Oak Trusted Runtime Authors (and its transitive dependencies)
```

Comment:
The "transitive dependencies" sounds weird here now. Do you mean our children? :)

Reply:
Reworded, this should also get better once we talk about Transparent Release here. cc @rbehjati